README.md

```diff
@@ -77,16 +77,25 @@ example-organization
 
 ### [5. appinfra](/5-appinfra/)
 
-The purpose of this stage is to create application-speciifc infrastructure, including the application CI/CD pipeline. This stage deploys the application-specific resources for the Cymbal Bank sample application, including service-specific databases. This stage also defines the CI/CD pipeline for each application, targeting the clusters in the multitenant infrastructure stage.
+The purpose of this stage is to create application-speciifc infrastructure, including the application CI/CD pipeline. This stage deploys the application-specific resources, including service-specific databases. This stage also defines the CI/CD pipeline for each application, targeting the clusters in the multitenant infrastructure stage.
```

Comment:
I wonder if we could be more generic than `databases`.  In the future it could conceivably be other types of resources.

Reply:
exactly!